editor/bundle-resources/x86_64-linux/defold.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# for testing purposes
+# function glxinfo() {
+# cat << EOF
+# server glx version string: 1.4
+# client glx version string: 1.4
+# GLX version: 1.4
+# Max core profile version: 4.6
+# Max compat profile version: 4.6
+# Max GLES1 profile version: 1.1
+# Max GLES[23] profile version: 3.2
+# OpenGL core profile version string: 4.6 (Core Profile) Mesa 20.2.6
+# OpenGL core profile shading language version string: 4.60
+# OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.2.6
+# OpenGL shading language version string: 4.60
+# OpenGL ES profile version string: OpenGL ES 3.2 Mesa 20.2.6
+# OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
+# GL_EXT_shader_implicit_conversions, GL_EXT_shader_integer_mix,
+# EOF
+# }
+
+
+if ! command -v glxinfo &> /dev/null
+then
+    echo "glxinfo could not be found"
+    echo "Install using 'sudo apt-get install mesa-utils'"
+    exit
+fi
+
+# detect MESA and GL version and set override
+export GL_VERSION=$(glxinfo | grep "OpenGL version string" | sed 's/OpenGL version string: \([0-9]*.[0-9]*\).*/\1/')
+
+# export MESA_VERSION=$(glxinfo | grep "OpenGL version string" | sed 's/OpenGL version string: .* Mesa \(.*\)/\1/')
+# export MESA_LOADER_DRIVER_OVERRIDE=i965
```

Comment:
I guess we should avoid commented out code, unless there's some documentation explaining it?